activation/poet.go

```diff
@@ -33,6 +34,12 @@ var (
 	ErrCertificatesNotSupported = errors.New("poet doesn't support certificates")
 )
 
+const (
+	submitPath = "/v1/submit"
+	infoPath   = "/v1/info"
+	proofPath  = "/v1/proofs"
+)
```

Comment:
What's the point of having constants used in a single place only? It increases the cognitive load when reading the code as one has to jump to the constant definition to see its value.